Event.h

```diff
@@ -36,11 +36,14 @@ class Event
   int  use_seeds_from_cmsswtracks(); //special mode --> use only seeds which generated cmssw reco track
   int  clean_cms_simtracks();
   int  clean_cms_seedtracks(TrackVec *seed_ptr = nullptr); //operates on seedTracks_; returns the number of cleaned seeds
+  int  clean_cms_seedtracks_iter(TrackVec *seed_ptr, const IterationConfig& itrcfg);
```

Comment:
Eventually we should move this out of Event (and probably remove the old seed_clean function and use the new one in the old iteration0 functions).
But we can do it in the next step, after we think through duplicate cleaning and params there.

Reply:
I prefer doing it as a next step